database/gorm/factory_test.go

```diff
@@ -0,0 +1,121 @@
+package gorm
+
+import (
+	"log"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/stretchr/testify/suite"
+
+	ormcontract "github.com/goravel/framework/contracts/database/orm"
+)
+
+type UserFactory struct {
+}
+
+func (u *UserFactory) Definition() any {
+	faker := gofakeit.New(0)
+	return map[string]interface{}{
+		"name":       faker.Name(),
+		"avatar":     faker.Email(),
+		"created_at": faker.Date(),
+		"updated_at": faker.Date(),
+	}
+}
+
+type FactoryTestSuite struct {
+	suite.Suite
+	factory *FactoryImpl
+	query   ormcontract.Query
+}
+
+func TestFactoryTestSuite(t *testing.T) {
+	suite.Run(t, new(FactoryTestSuite))
+}
+
+func (s *FactoryTestSuite) SetupTest() {
+	mysqlDocker := NewMysqlDocker()
+	_, _, mysqlQuery, err := mysqlDocker.New()
+	if err != nil {
+		log.Fatalf("Init mysql error: %s", err)
+	}
```

Comment:
S: Sorry, forgot this, we can move this logic to `TestFactoryTestSuite`(because every test case will create a MySQL docker if the logic is here), and don't forget to release docker: `assert.Nil(t, mysqlPool.Purge(mysqlResource))`.

Just like:
<img width="1476" alt="image" src="https://github.com/goravel/framework/assets/24771476/2c69b3c4-49a9-463e-b61d-4aec44d06ade">